dist/stat_data.py

```diff
@@ -731,8 +731,10 @@ def __init__(self, name, desc, flags=''):
     TxnStat('txn_set_ts', 'set timestamp calls'),
     TxnStat('txn_set_ts_durable', 'set timestamp durable calls'),
     TxnStat('txn_set_ts_durable_upd', 'set timestamp durable updates'),
+    TxnStat('txn_set_ts_force', 'set timestamp force calls'),
     TxnStat('txn_set_ts_oldest', 'set timestamp oldest calls'),
     TxnStat('txn_set_ts_oldest_upd', 'set timestamp oldest updates'),
+    TxnStat('txn_set_ts_out_of_order', 'set timestamp resulted in out-of-order timestamps'),
```

Comment:
I think it would be better if the text was more specific, something like: "set timestamp global oldest timestamp set to be more recent than the stable timestamp"

Reply:
Updated in f3a3be32b66679cafbec5f5fe0db68701c91923a